docs/model/rules-to-model.md

```diff
@@ -42,7 +47,7 @@ Units of model without persisted entity:
 * Heureka
 * ShopInfo
 
-## Model grids
+### Model grids
```

Comment:
are you sure that `## MultidomainEnityClassProvider` is outside this group ?

Reply:
yeah, these are exceptions to the model rules. MultidomainEntityClassProvider is in my opinion different story.